LocationReminders/ViewController.m

```diff
@@ -8,20 +8,20 @@
 
 #import "ViewController.h"
 #import "AddReminderViewController.h"
+#import "LocationControllerDelegate.h"
+#import "LocationController.h"
 
 @import Parse;
 @import MapKit;
 @import CoreLocation;
 
-@interface ViewController () <CLLocationManagerDelegate, MKMapViewDelegate>
+@interface ViewController () <CLLocationManagerDelegate, MKMapViewDelegate, LocationControllerDelegate>
```

Comment:
You don't need this to conform to the `CLLocationManagerDelegate` any more --  your `LocationManager` is the class that should be conforming to that delegate.